src/js/NavAccordion.js

```diff
@@ -1,54 +1,71 @@
-(function($) {
+(function() {
 
-	var navAccordion = function( $el ) {
+	var navAccordion = function( el ) {
 
-		$el.children().each( function() {
+		Array.prototype.forEach.call( el.children, function( navAccordion_Item ) {
 
-			var $NavAccordion_Item = $(this);
-			var $subNav = $NavAccordion_Item.children( 'ul' );
+			var subNav = Array.prototype.filter.call( navAccordion_Item.children, function( el ) {
+				return el.tagName === 'UL';
+			} );
 
-			if ( $subNav.length > 0 ) {
+			if ( subNav.length < 1 ) {
+				return;
+			} else {
+				subNav = subNav[0];
+			}
+
+			var openHeight = subNav.offsetHeight;
+			var style      = getComputedStyle( subNav );
 
-				var openHeight           = $subNav.outerHeight();
-				var $NavAccordion_Toggle = $( '<button/>', {
-					text: 'Toggle',
-					class: 'Btn NavAccordion_Toggle'
-				} );
+			openHeight += parseInt( style.marginTop ) + parseInt( style.marginBottom );
 
-				var toggleSubNav = function( show ) {
+			var navAccordion_Toggle = document.createElement( 'BUTTON' );
+			navAccordion_Toggle.appendChild( document.createTextNode( "Toggle" ) );
+			navAccordion_Toggle.classList.add( 'Btn' );
```

Comment:
[CanIUse.com](http://caniuse.com/#search=classlist) suggest that IE11 doesn't support multiple params being passed to `classList.add`